clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -156,7 +156,8 @@ Changes in existing checks
 
 - Improved :doc:`cppcoreguidelines-missing-std-forward
   <clang-tidy/checks/cppcoreguidelines/missing-std-forward>` check by no longer
-  giving false positives for deleted functions.
+  giving false positives for deleted functions and fix false negative when one
+  parameter is forwarded, but some other parameter isn't.
```

Comment:
```suggestion
  giving false positives for deleted functions and fix false negative when some
  parameters are forwarded, but other aren't.
```

Reply:
Thanks for review to make the description more clear!
I think you can check https://github.com/llvm/llvm-project/blob/2a1b09fee4b4b22f4f7189695ce3e858b91a8d69/clang-tools-extra/test/clang-tidy/checkers/cppcoreguidelines/missing-std-forward.cpp#L171-L173
this testcase may helpful for solving confusions.